chapter08/sxg/ex_8_2.pl

```diff
@@ -0,0 +1,19 @@
+#!perl
+use warnings;
+use strict;
+
+my $input;
+
+if ( @ARGV == 1 ){

```

Comment:
You don't need to change the program to look at argv, you could just redirect a file into its STDIN